plugins/Sf2Player/Sf2Player.cpp

```diff
@@ -537,6 +545,38 @@ void  Sf2Instrument::updateChorus()
 
 
 
+void Sf2Instrument::updateTuning()
+{
+	if (instrumentTrack()->microtuner()->enabledModel()->value())
+	{
+		auto centArray = std::array<double, 128>{};
+		double lowestHz = powf(2.f, -69.f / 12.f) * 440.f;// Frequency of MIDI note 0, which is approximately 8.175798916 Hz
+		for (int i = 0; i < 128; ++i)
+		{
+			// Get desired Hz of note
+			centArray[i] = instrumentTrack()->microtuner()->keyToFreq(i, DefaultBaseKey);
+			// Convert Hz to cents
+			centArray[i] = 1200.f * log2(centArray[i] / lowestHz);
+		}
```

Comment:
It seems a bit odd to store everything as `double`s, but do the computation with `float`s. I would remove the `f` suffixes from the literals, and use `std::pow` instead. Also, I think it would be clearer to use a temporary variable for the note frequency, rather than using `centArray[i]` for two different things.